Mlem/Views/Tabs/Settings/Components/Views/Appearance.swift

```diff
@@ -149,7 +152,23 @@ struct AppearanceSettingsView: View {
                     isTicked: $shouldShowCommunityIcons
                 )
             }
-            
+            Section("Navigation")
+            {
+                SwitchableSettingsItem(
+                    settingPictureSystemName: "scroll.fill",
+                    settingPictureColor: .pink,
+                    settingName: "Hide TopBar and NavBar when scrolling",
```

Comment:
Should we reword this to _"Hide top and bottom bars while scrolling"_, `TopBar` and `NavBar` sound like variable names 😉

Reply:
done, thanks!